### DISCUSSION TITLE
I think I am missing a setting to make my solution have a proper chain and be considered vendor signed

### DISCUSSION THREAD

**Discussion by gleneccles** [polymind-inc/acmebot#562] [Q&A] [answered]
First of all, this was easy to set up and it is making certs which is pretty awesome!

Where I am getting hung up is when I am running some SSL checker/scanner websites. I am getting different results than other Let's Encrypt websites.

This is the website: https://www.sslchecker.com/sslchecker

For example, my site (www.gleneccles.com) says it is not vendor signed and that there is chain information missing. Other websites like shop.bbc.com or nsa.gov are returning jut fine.

I feel like I have it all set up right, but I must be missing something. I have it set up with `Acmebot:PreferredChain = ISRG Root X1` thinking that might help, but no change. I am using Azure DNS.

**Comment by shibayan** [comment, marked as answer]
Using Qualys to check, I can confirm that the certificate chain is built without any problems.
https://www.ssllabs.com/ssltest/analyze.html?d=www.gleneccles.com

I believe it is a compatibility with the App Service, because sslchecker gives the same warning when the default domain of Azure Web Apps is entered.

**Reply by gleneccles** [reply]
Hey! Thanks so much for your reply and for this tool. Much appreciated. Have a great day!
